samples/java-protobuf-customer-registry-kafka-quickstart/src/it/java/customer/api/CustomerActionIntegrationTest.java

```diff
@@ -51,7 +53,7 @@ public void createAndPublish() throws Exception {
     createCustomer(customer);
 
     // wait for action to publish the change of state
-    var createdMsgOut = outTopic.expectOneTyped(CustomerApi.Customer.class);
+    var createdMsgOut = outTopic.expectOneTyped(CustomerApi.Customer.class, Duration.of(10, ChronoUnit.SECONDS));
```

Comment:
There is another usage of `outTopic.expectOneTyped` a few lines below. 
I might want to also increase its timeout.

Reply:
https://github.com/lightbend/kalix-jvm-sdk/pull/1875/files#diff-94774a57e19f6ef59ee19111961b18d2a67321eb59c93c3e6869bbf39b8401d7R73

Reply:
I assumed the flakiness come from combining the boostrapping of this on the runtime side + the delay on publishing the state, thus most likely to happen on the first call than subsequent ones.. But yeah, I would guess it can also happen in a second call.